lib/ewasm/system/transform-i64.wast

```diff
@@ -0,0 +1,13 @@
+(module
+  (import "interface" "useGas" (func $useGas (param i32 i32)))
+
+  (export "useGas" (func $useGasShim))
```

Comment:
There's a lot more shims needed. Can you add a comment into this file stating the purpose and that this only implement those currently used, but not all the EEI methods?